tests/TotalPriceCalculation.spec.ts

```diff
@@ -30,26 +23,63 @@ test('navigate to chairs page', async ({page})=>{
         await onLightingPage.scrollToProductElement()
 
         //click on the product
-        await onLightingPage.ClickOnAdjustableWallLampProduct();
+        await onLightingPage.clickOnAdjustableWallLampProduct();
+       
+        //Get the value of the first element.
+        const firstPrice = await page.locator('.price').innerText() 
+        console.log("FIRST PRICE:", firstPrice)
```

Comment:
If you want to log something that needs to be displayed to the reader, adjust name of the log.